CMakeLists.txt

```diff
@@ -84,7 +84,16 @@ option(HLSL_ENABLE_FIXED_VER "Sets up fixed version information." OFF) # HLSL Ch
 option(HLSL_ENABLE_ANALYZE "Enables compiler analysis during compilation." OFF) # HLSL Change
 option(HLSL_OPTIONAL_PROJS_IN_DEFAULT "Include optional projects in default build target." OFF) # HLSL Change
 
-option(ENABLE_SPIRV_CODEGEN "Enables the HLSL to SPIR-V conversion flow." OFF) # SPIRV Change
+# SPIRV change starts
+option(ENABLE_SPIRV_CODEGEN "Enables SPIR-V code generation." OFF)
+option(SPIRV_BUILD_TESTS "Build targets for the SPIR-V unit tests." OFF)
+if ( ${SPIRV_BUILD_TESTS})
```

Comment:
Fixed. Interesting thing is that `if()` has two different formatting ways in `CMakeLists.txt` itself, one with spaces within parentheses, one without.